tests/test_interface.py

```diff
@@ -74,12 +55,24 @@ def test_theme_apply_to():
     lexer.setEolFill = mock.MagicMock(return_value=None)
     lexer.setPaper = mock.MagicMock(return_value=None)
     theme.apply_to(lexer)
-    assert lexer.setFont.call_count == 2
+    assert lexer.setFont.call_count == 17
```

Comment:
This used to be 2 because a new font was only added to the editor if/when it was actually different from the base font.  

I've made the assumption here that calling setFont with the same font won't be a measurable performance impact, so just made it call setFont each time.  

Re-implementing the old behaviour wouldn't be too tricky here, if that's better